storage.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	// "bufio"
+	"context"
+	"log"
+	"net"
+	"os"
+	"strings"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+var dbPool *pgxpool.Pool
+
+func initDb() {
+	ctx := context.Background()
+
+	var err error
+	dbPool, err = pgxpool.Connect(ctx, os.Getenv("DATABASE_URL"))
+	checkError("unable to connect to the database", err)
+
+	schemaBytes, err := os.ReadFile("sql/schema.sql")
+	checkError("error reading `sql/schema.sql`", err)
+
+	_, err = dbPool.Exec(ctx, string(schemaBytes))
+	checkError("failed to create the database schema", err)
+}
+
+func checkBlocklist(ipAddress net.IP) bool {
```

Comment:
If found returns `true`?

Reply:
Thank you for the comment. Yes. But now I realize that the function name is not the best. Let me update it to `inBlocklist`. (Unless you have a more idiomatic suggestion?)

Reply:
I am already using `inBlocklist`. I will just name it `isIPAddressInBlocklist`. Or is it more idiomatic to create a new module, say `storage`, and call it whatever in there. ... That actually seems better. But I also like the flatness currently.